spring-boot-project/spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -444,21 +444,22 @@ recommend that you add it to your primary `@Configuration` class.
 
 
 [[using-boot-replacing-auto-configuration]]
-=== Gradually replacing auto-configuration
-Auto-configuration is noninvasive,  at any point you can start to define your own
+=== Gradually Replacing Auto-configuration
+Auto-configuration is noninvasive. At any point, you can start to define your own
 configuration to replace specific parts of the auto-configuration. For example, if
-you add your own `DataSource` bean, the default embedded database support will back away.
+you add your own `DataSource` bean, the default embedded database support backs away.
 
 If you need to find out what auto-configuration is currently being applied, and why,
-start your application with the `--debug` switch. This will enable debug logs for a
-selection of core loggers and log an auto-configuration report to the console.
+start your application with the `--debug` switch. Doing so enables debug logs for a
+selection of core loggers and logs an auto-configuration report to the console.
 
 
 
 [[using-boot-disabling-specific-auto-configuration]]
-=== Disabling specific auto-configuration
-If you find that specific auto-configure classes are being applied that you don't want,
-you can use the exclude attribute of `@EnableAutoConfiguration` to disable them.
+=== Disabling Specific Auto-configuration Classes
+If you find that specific auto-configure classes that you do not want are being applied,
```

Comment:
Not your doing but I think this should be "auto-configuration" rather than "auto-configure"

Reply:
Good catch, but I think the better fix is to change "auto-configure" to "auto-configuration" in the first sentence. "auto-configure" should be a verb (that gets fuzzy in a hurry, though), while "auto-configuration should be the noun and adjective. So it's a smidgeon more correct to write, "auto-configuration."